### PR TITLE
Setup docker

### PR DESCRIPTION
**Pull request by xModulus**
Help

### CODE REVIEW

**Comment by iamfeek** [comment]
@xModulus are we using postgres 9.6? 

Lets use latest version?
11 has LTS, IIRC.
(12 the is the current version)

**Comment by xModulus** [comment]
> @xModulus are we using postgres 9.6?
> 
> Lets use latest version?
> 11 has LTS, IIRC.
> (12 the is the current version)

I didn't know which version to use, just took the first one I saw on dockerhub.  So postgres:11-alpine eh?

File: env.sample [hunk 1]
```diff
@@ -1 +1,12 @@
 BOT_TOKEN=
+
+#HOST=localhost
+#PORT=8000
+#NODE_ENV=development
+#LOG_LEVEL=debug
```
[the remark this answers is not in the document]
Comment (xModulus) [reply env.sample]: They're not in use at the moment from what I can tell.

[the remark this answers is not in the document]
Comment (ruqqq) [reply env.sample]: I mean you added these right? If not in use should we remove them totally?

File: ormconfig.js [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
-  port: 5432,
-  username: 'postgres',
-  password: 'postgres',
-  database: 'postgres',
+  type: process.env.DB_TYPE,
+  host: process.env.POSTGRES_HOST,
+  port: process.env.POSTGRES_PORT,
+  username: process.env.POSTGRES_USERNAME,
+  password: process.env.POSTGRES_PASSWORD,
+  database: process.env.TPOSTGRES_DATABASE,
```
[the remark this answers is not in the document]
Comment (xModulus) [reply ormconfig.js]: Yup :P

File: env.sample (hunk 1 above)
[the remark this answers is not in the document]
Comment (xModulus) [reply env.sample]: I just used a template from my other node projects. Sure I'll remove them

**Pull request merged**